app/services/pii/password_encryptor.rb

```diff
@@ -1,29 +1,17 @@
 module Pii
   class PasswordEncryptor < ::Pii::Encryptor
-    def initialize
-      self.encrypted_key_maker = EncryptedKeyMaker.new
-      super
-    end
-
+    # TODO: Refactor this class to be initialized with a user access key :reek:FeatureEnvy
```

Comment:
Doing this makes a bit of a big diff, so I've got a follow on PR coming that does it to keep the PR reviewable.